components/FlexibleHeader/src/MDCFlexibleHeaderView.m

```diff
@@ -824,14 +833,6 @@ - (void)fhv_accumulatorDidChange {
 
 - (void)fhv_updateLayout {
   if (!_trackingScrollView) {
-    // Even if we're not tracking a scroll view, our minimumHeight might have changed due to new
-    // safe area insets.
-    if (_minimumHeight > 0 && CGRectGetHeight(self.bounds) != _minimumHeight) {
```

Comment:
Are we sure we want to remove this logic?

Reply:
Spoke offline: this logic was introduced last week in order to resolve safe area insets bugs. This PR moves it to the more relevant location.

Reply:
Yup, just moving what I added in #2068 to a more relevant location to prevent this code from running when we don't need it to.